docs/Installation/Installation.md

```diff
@@ -31,10 +31,10 @@ all of these dependencies.
 * [Charm++](http://charm.cs.illinois.edu/) 6.10.2
 * [Git](https://git-scm.com/)
 * BLAS (e.g. [OpenBLAS](http://www.openblas.net))
-* [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.7
+* [Blaze](https://bitbucket.org/blaze-lib/blaze/overview) v3.8
 * [Boost](http://www.boost.org/) 1.60.0 or later
 * [Brigand](https://github.com/edouarda/brigand)
-* [Catch](https://github.com/philsquared/Catch) 2.8.0 or later
+* [Catch](https://github.com/philsquared/Catch) 2.8.0 or later (it is recommended to use single-header installation)
```

Comment:
- Would you mind updating the link to `https://github.com/catchorg/Catch2`?
- I'm not so sure we should add this recommendation. For instance, for me it was easiest to install catch through my package manager (macOS). On Minerva I installed it with Spack. I think we can't give a general recommendation here how to install things, but only on the installation options (like "disable mpi" for HDF5 and libsharp below). Which issue did you run into that prompted you to add this note? If you feel like adding a note to help others with that issue, perhaps phrase it more specific to that issue?

Reply:
- The current link redirects you to Catch2, but I'll still update it
- I was installing all the dependencies on my machine and inside catch there is `include/catch.cpp` and `single_include/catch2/catch.hpp`. I tried the first option but got a lot of errors when compiling about missing headers. The second option (single include) is what we are supposed to use and what is used inside the containers I believe.
- When (if) we switch to v3, then we can update docs. I don't think that should matter for this because it's v2.

Reply:
Their `devel` branch is already in v3 and has a bit different source structure with v2.x. Should we add a phrase like '2.8.0 or later but not 3.x'?

Reply:
As long as we don't yet support v3, then yeah I think that's ok.

Reply:
ok the way you formulated this looks fine to me. If you want to include @yoonso0-0's suggestion to exclude v3 then squash that in as well. No need to wait for my re-approval if someone gets to review this before me.